modules/acquia_cms_page/config/optional/node.type.page.yml

```diff
@@ -1,17 +1,23 @@
+uuid: b8e80b75-8f73-4536-9b41-b25265bfae22
 langcode: en
 status: true
 dependencies:
+  module:
+    - acquia_cms_common
+    - menu_ui
+    - node_revision_delete
```

Comment:
node revision delete is not a dependency in page, remove this

Reply:
changes are done